radio/apps/schedules/models.py

```diff
@@ -285,20 +272,17 @@ def schedule(cls, dt, exclude=None):
     @classmethod
     def get_next_date(cls, programme, after):
         list_schedules = cls.objects.filter(programme=programme, type='L')
-        list_schedules = list_schedules.filter(
-            programme__end_date__isnull=True
-        ) | list_schedules.filter(
-            programme__end_date__gte=after
+        list_schedules = (
+            list_schedules.filter(programme__end_date__isnull=True) |
+            list_schedules.filter(programme__end_date__gte=after)
```

Comment:
Make code more readable. The important part is to apply multiple filters, so the code should not spread this across multiple indentation levels, but should reflect this by intuitive reading.